src/js/backup_restore.js

```diff
@@ -412,13 +411,14 @@ function configuration_restore(callback) {
 
             // LED Strip was saved as object instead of array.
             if (typeof(configuration.LED_STRIP) == 'object') {
-                var fixed_led_strip = [];
+                const fixedLedStrip = [];
 
-                var index = 0;
+                let index = 0;
                 while (configuration.LED_STRIP[index]) {
-                    fixed_led_strip.push(configuration.LED_STRIP[index++]);
+                    fixedLedStrip.push(configuration.LED_STRIP[index]);
+                    index++;
```

Comment:
Not sure why you are changing this - one of the niceties you get from the increment operator is elegant index looping.

Reply:
This was also incompliant with sonarcloud and had to fix it in this way

Reply:
:disappointed: SonarCloud has weird ideas some times...

Reply:
Yeah tried using `for (of)` instead of `for (in)` (or vice versa) in some cases and Soundcloud want's the other way is counter intuitive.